src/database/DataTypes/Item.js

```diff
@@ -168,6 +168,18 @@ export class Item extends Realm.Object {
     return this.unit?.units ?? generalStrings.not_available;
   }
 
+  /**
+   * @return {Date} The date of the most recent confirmed requisition for this item.
+   */
+  get lastRequisitionDate() {
+    const mostRecentRequisitionItem = this.requisitionItems.sorted(
```

Comment:
should this perhaps be filtering for confirmed reqs? The comment indicates that we should expect confirmed only.
should it sort by `confirmDate` rather than `entryDate`?